deepfence_frontend/apps/dashboard/src/features/postures/pages/PostureCloudDetailModal.tsx

```diff
@@ -69,7 +69,7 @@ const Header = () => {
 };
 
 function processLabel(labelKey: string) {
-  return labelKey.replaceAll('_', ' ').replaceAll('id', 'ID');
+  return labelKey.replaceAll('_', ' ').replace(new RegExp('\\bid', 'g'), 'ID');
```

Comment:
use functions instead of regex. see metadata.tsx in topology. 

in the metadata.tsx also use `===` instead of `.includes()`